src/plugins/pbjx/components/revert-modal/index.jsx

```diff
@@ -0,0 +1,114 @@
+import PropTypes from 'prop-types';
```

Comment:
@rsumilang got some eslint jazz going on in this component

Reply:
![Screen Shot 2020-06-09 at 2 26 55 PM](https://user-images.githubusercontent.com/1748115/84203082-b09d4a00-aa5d-11ea-9003-273a90589bd4.png)

I don't see any.

Reply:
<img width="1044" alt="Screen Shot 2020-06-09 at 3 45 11 PM" src="https://user-images.githubusercontent.com/6315388/84208278-3920e800-aa68-11ea-85a2-32cd9aef187c.png">